### PR TITLE
Cleanup visualizations

### PR DESCRIPTION
**Pull request by mejarrett**
This prevents the `shards failed, too_many_buckets_exception` errors on visualizations that we know may return over 10,000 buckets (typically seen with the time picker set to the last 2 hours, or longer). We also removed a duplicated visualization (`Top-10-Dest-Ports-By-Flow-Count`) in favor of `Top-10-Dest-Ports-(pie)`.

Changes:
- Removed the Top-10-Dest-Ports-By-Flow-Count visualization

- Error prevention: On the Destination Port Dashboard and Replayed Traffic Dashboard, the `Top 10 Dest Ports (Pie)` donut chart has been reduced to two rings showing the top 10 destination ports and the top 10 applications on each port. The source and destination IP addresses are still available in the Ports Table at the bottom of the page. 

- Error prevention: We removed `Sum of Repeated Field Count by Application` on the Traffic Profile dashboard. In the past, this was interesting troubleshooting information (not useful to users as far as we know) because we used to insert every value on a repeated field into ES, so high repeated field count could cost us a lot of metadata space. Since we now truncate repeated fields over X length prior to insert, this is no longer useful for troubleshooting.

### CODE REVIEW

File: resources/visualization:Replayed-Traffic-Dashboard-Description.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
    "migrationVersion": {
-      "visualization": "7.2.0"
+      "visualization": "7.4.2"
```
Comment (craig-cogdill) [inline resources/visualization:Replayed-Traffic-Dashboard-Description.json:3]: Why did all of these change to 7.4.2? Those changes seem like they should be reverted.

File: resources/dashboard:Destination-Port-Dashboard.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
    "migrationVersion": {
-      "dashboard": "7.0.0"
+      "dashboard": "7.3.0"
```
Comment (craig-cogdill) [inline resources/dashboard:Destination-Port-Dashboard.json:3]: And this one is 7.3.0 for some reason?

**Review by craig-cogdill — CHANGES_REQUESTED** [review]

**Review by craig-cogdill — APPROVED** [review]

**Comment by craig-cogdill** [comment]
Approved changes after Slack discussion, but I have not built or tested this.

**Review by Schatzman — APPROVED** [review]
Built and tested 👍

**Review by josh-cogdill — APPROVED** [review]

**Pull request merged**